README.md

```diff
@@ -46,6 +46,69 @@ Once the project has fully initialized, you can start the project from now on us
 
 We have designed the aggregator project (potentially moved to API) to initialize the database for you, it also triggers an "instance" of your choosing via code so you're able to quicky start tracking data. Check it's readme for more information.
 
+
+## Adding certificates for local dev HTTPS access
+
+If you want to access the dev environment over HTTPS (Say, because Chrome forces it), you'll need to follow some additional steps:
+<details>
+    <summary>Expand certificate instructions...</summary>
+
+1. The following needs to be added to `traefik/config/config.yml`
+    ```yaml
+    tls:
+      stores:
+        default:
+          defaultCertificate:
+            certFile: /etc/certs/api.dev.ps2alerts.com.pem
+            keyFile: /etc/certs/api.dev.ps2alerts.com-key.pem
+      certificates:
+      - certFile: /etc/certs/api.dev.ps2alerts.com.pem
+        keyFile: /etc/certs/api.dev.ps2alerts.com-key.pem
+      - certFile: /etc/certs/dev.ps2alerts.com.pem
+        keyFile: /etc/certs/dev.ps2alerts.com-key.pem
+      - certFile: /etc/certs/wss.dev.ps2alerts.com.pem
+        keyFile: /etc/certs/wss.dev.ps2alerts.com-key.pem
+    ```
+2. The referenced certificates must be generated and placed in the ~/ps2alerts/certs/ directory. To generate **local** self-signed certs:
+   Steps generally referenced from: https://www.section.io/engineering-education/how-to-get-ssl-https-for-localhost/
+    - `cd ~/ps2alerts/certs`
+    - `openssl genrsa -out CA.key -des3 2048`
+    - `openssl req -x509 -sha256 -new -nodes -days 3650 -key CA.key -out CA.pem`
```

Comment:
`Common Name (e.g. server FQDN or YOUR name) []:Ryan`

I just used my first name when generating the `CA.pem` file and everything worked for me. Is MacOS more stringent about that? 

As far as I know I think its fine to use whatever, its just about trusting the created root certificate.

I'll put a note about being consistent between the two certs for country code and all

Reply:
Now that I'm adding the macOS note I think the difference was that you trusted the `dev.ps2alerts.com.pem` certs directly rather than adding `CA.pem` as a trusted root certificate